endpoints/bahmni/input-validation.json

```diff
@@ -1,553 +1,203 @@
         }
       }
     },
-    "attributes": {
-      "type": "array",
-      "element": {
-        "display": {
-          "type": "string"
-        },
-        "uuid": {
-          "type": "string"
-        },
-        "value": {
-          "type": "string"
-        },
-        "attributeType": {
-          "element": {
-            "uuid": {
-              "type": "string"
-            },
-            "display": {
-              "type": "string"
-            },
-            "links": {
-              "type": "object",
-              "rel": {
+    "person": {
+      "type": "object",
+      "properties": {
+        "names": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "display": {
                 "type": "string"
               },
-              "uri": {
+              "familyName": {
+                "type": "string"
+              },
+              "middleName": {
+                "type": "string"
+              },
+              "givenName": {
                 "type": "string"
               }
             }
           }
         },
+        "birthdate": {
+          "type": "string",
+          "format": "date-time"
+        },
         "voided": {
           "type": "boolean",
           "default": false
         },
-        "links": {
-          "type": "array",
-          "element": {
-            "rel": {
-              "type": "string"
-            },
-            "uri": {
-              "type": "string"
-            }
-          }
+        "dead": {
+          "type": "boolean",
+          "default": false
         },
-        "resourceVersion": {
+        "deathDate": {
           "type": "string",
-          "required": true
-        }
-      }
-    },
-    "voided": {
-      "type": "boolean",
-      "default": false
-    },
-    "auditInfo": {
-      "type": "object",
-      "element": {
-        "creator": {
-          "uuid": {
-            "type": "string"
-          },
-          "display": {
-            "type": "string"
-          },
-          "links": {
-            "type": "array",
-            "element": {
-              "rel": {
+          "format": "date-time"
+        },
+        "addresses": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "display": {
+                "type": "string"
+              },
+              "cityVillage": {
+                "type": "string"
+              },
+              "stateProvince": {
+                "type": "string"
+              },
+              "country": {
                 "type": "string"
               },
-              "uri": {
+              "postalCode": {
                 "type": "string"
+              },
+              "countyDistrict": {
+                "type": "string"
+              },
+              "startDate": {
+                "type": "string",
+                "format": "date-time"
```

Comment:
@bradsawadye Are you sure of the format of this field?

```suggestion
                "nullable": true
```

Reply:
fixed in https://github.com/jembi/openhim-mediator-mapping/pull/21/commits/da90c901b05f8fcd2480d722da8ccaa7f6a86d77